### PR TITLE
Several updates

### PR DESCRIPTION
**Pull request by geeknoid**
- Populate _data/archives.yml into the sites since that's what they use to create
the version-selection dropdown.

- Stop post-editing the generated sites since we will leverage the site archive mode
in the first place.

- Add a site-wide search engine.

- Add a robots.txt file

### CODE REVIEW

File: archive.istio.io/public/404.html [hunk 1]
```diff
@@ -13,42 +13,50 @@
...
-    <link rel="apple-touch-icon" href="/v0.5/favicons/apple-touch-icon-180x180.png" sizes="180x180">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/favicon-16x16.png" sizes="16x16">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/favicon-32x32.png" sizes="32x32">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-36x36.png" sizes="36x36">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-48x48.png" sizes="48x48">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-72x72.png" sizes="72x72">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-96x196.png" sizes="96x196">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-144x144.png" sizes="144x144">
-    <link rel="icon" type="image/png" href="/v0.5/favicons/android-192x192.png" sizes="192x192">
+    <link rel="shortcut icon" href="./v0.5/favicons/favicon.ico">
```
Comment (ldemailly) [inline archive.istio.io/public/404.html]: this can’t have ./ - it won’t work as a 404 that way - see my previous fix to it

**Comment by geeknoid** [comment]
*sigh*.

Fixed. This would be easier if I could actually build and test things :-(

**Comment by ldemailly** [comment]
you didn’t get ‘firebase serve’ to work ?

**Comment by geeknoid** [comment]
Ready for checkin.

**Comment by ldemailly** [comment]
ahem... unless there is some emergency, can you make PRs (you could have updated this one, it was accepted and merged it...) ?

https://github.com/istio/admin-sites/commit/629352eff2de49476af15b86c569a29d1474d81b
https://github.com/istio/admin-sites/commit/b7c548858935db47d00f685573d8e2d14fab157b

**Pull request closed without merging**